cart/cart.py

```diff
@@ -92,3 +93,16 @@ def clear(self):
         for item in self.cart.item_set.all():
             item.delete()
 
+    def is_empty(self):
+        return self.count() == 0
+
+    def cart_serializable(self):
```

Comment:
Awesome method by I don't see an automated test to validate it. Please add tests and I'll merge these. Thanks for the contribution!